conf/test.config

```diff
@@ -22,6 +22,7 @@ params {
   flowcell = 'FLO-MIN106'
   kit = 'SQK-DCS109'
   barcode_kit = 'EXP-NBD103'
+  skip_quantification = true
```

Comment:
Is there a reason why this is skipped?

Reply:
I can't test it locally so just removed them here https://github.com/nf-core/nanoseq/pull/89/commits/45b9d02f5e2371a84e816305de4305dafbe9fa40 to see if the tests still pass.

Reply:
The [samplesheet](https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/samplesheet_bc_dx.csv) doesn't have the same genome entries so we need to keep that in here or we get the error below which is expected:

```
Quantification can only be performed if all samples in the samplesheet have the same reference fasta and GTF file."
                   Please specify the '--skip_quantification' parameter if you wish to skip these steps.
```